homeassistant/components/sensor/rest.py

```diff
@@ -68,20 +72,25 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     rest = RestData(method, resource, auth, headers, payload, verify_ssl)
     rest.update()
 
-    add_devices([RestSensor(hass, rest, name, unit, value_template)], True)
+    add_devices([RestSensor(hass, rest, name, unit,
+                            value_template, json_attrs)])
```

Comment:
Revert the True at the end

Reply:
Sorry about that. The flag was added after I forked the code and I missed that in the merge.